### DISCUSSION TITLE
Creating Screen

### DISCUSSION THREAD

**Discussion by George-Fx** [orchidsoftware/platform#2224] [Q&A]
I'm trying to create a screen with the command: php artisan orchid:screen Idea
But a file is created with a different code than the one specified in the documentation.

Here is what the team created for me:

`<?php

namespace App\Orchid\Screens;

use Orchid\Screen\Screen;

class Idea extends Screen
{
    /**
     * Query data.
     *
     * @return array
     */
    public function query(): iterable
    {
        return [];
    }

    /**
     * Display header name.
     *
     * @return string|null
     */
    public function name(): ?string
    {
        return 'Idea';
    }

    /**
     * Button commands.
     *
     * @return \Orchid\Screen\Action[]
     */
    public function commandBar(): iterable
    {
        return [];
    }

    /**
     * Views.
     *
     * @return \Orchid\Screen\Layout[]|string[]
     */
    public function layout(): iterable
    {
        return [];
    }
}
`
And this is the code from the documentation:
![Screenshot_16](https://user-images.githubusercontent.com/64652126/167265094-14471da8-b296-4f4a-8fd2-7f5328883458.png)

**Comment by mksnyder57** [comment]
You need to add the name you want, and add the description block with the description you want.  Add it in exactly as the documentation recommends and test by pointing your browser to URL/idea.  Then change up the name or description and view the changes by refreshing that page
